src/components/shared/Icon/Info.tsx

```diff
@@ -1,8 +1,8 @@
 import { FC } from 'react';
 import { InnerIconProps, withIcon } from 'tempus-ui';
 
-const Close: FC<InnerIconProps> = ({ size, color }) => {
-  const selectedColor = color || 'var(--infoColor, var(--textSecondary))';
+const Close: FC<InnerIconProps> = ({ size }) => {
+  const color = 'var(--infoColor, var(--textSecondary))';
```

Comment:
set --infoColor variable in scss

we now store all colors as scss variables, so it's easier to just define colors in scss instead of setting them on a component using color prop.